x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.scss

```diff
@@ -0,0 +1,11 @@
+.actAlertsList__tableRowDisabled {
+  background-color: $euiColorLightestShade;
+
+  .actAlertsList__tableCellDisabled {
+    color: $euiColorDarkShade;
+  }
+
+  .euiLink + .euiIcon {
+    margin-left: $euiSizeXS;
+  }
```

Comment:
I think we can put the tooltip message on the name itself and just remove the icon altogether—avoiding this specific css

Reply:
<img width="1394" alt="Screen Shot 2020-12-10 at 10 38 31 AM" src="https://user-images.githubusercontent.com/55110838/101815405-55afe280-3ad4-11eb-9559-1b5a528c127b.png">